README.md

```diff
@@ -0,0 +1,39 @@
+The Workiva Styleguide
+======================
+
+This repository is a collection of coding standards implemented by Workiva. They are provided here as a free resource to help other members of the development community, and are always open to suggestions on how they can be improved.
+
+\[A Koan\](<http://thecodelesscode.com/case/94?topic=compromise>)
```

Comment:
Did you want this to display as `[A Koan](http://thecodelesscode.com/case/94?topic=compromise)` or were you intending this to be a link with the text `A Koan`?